cmd/veneur/main.go

```diff
@@ -2,11 +2,10 @@ package main
 
 import (
 	"flag"
-	"net"
 	"sync"
 	"time"
 
-	log "github.com/Sirupsen/logrus"
+	"github.com/Sirupsen/logrus"
```

Comment:
Huh, logrus sounds really neat. Do the logs come out the same? (I'm wondering about aggregation mostly)